src/tryceratops/files/discovery.py

```diff
@@ -53,7 +53,7 @@ def _parse_python_files_from_dir(self, dir: str) -> Generator[ParsedFileType, No
             try:
                 parsed, filefilter = parse_file(filename)
             except Exception as ex:
-                logger.exception(f"Failed to parse file {filename}, skipping it")
+                logger.debug(f"Failed to parse file {filename}, skipping it")
```

Comment:
Maybe you should add `str(ex)` so we know the why.

Logging as `debug` won't print stack trace. I still wonder if that's what we want...
Let's decide later before merging, I'll test it as well so we can discuss more.

Reply:
I didn't really get too much into updating the message to print to stdout but I think having the file and line number are bare minimum.